### DISCUSSION TITLE
Scanning frequency "0" ???

### DISCUSSION THREAD

**Discussion by kx1t** [rtl-airband/RTLSDR-Airband#326] [Q&A]
I implemented [this rtl_airband.conf](https://termbin.com/w5xo) and when I look at the scanning display, it also scans through a frequency`0` ...
![image](https://user-images.githubusercontent.com/15090643/184213416-1e807c26-df20-4565-9692-30daae6fdb41.png)

(sorry for the humongous size screenshot!)

It wouldn't be much of an issue, but sometimes the squelch opens up on this frequency. Obviously, there is no frequency `0` that I would want to include. Any idea what I am doing wrong?

(Note -- the screenshot doesn't show this, but it scans through the rest of the frequencies without any issues.)

Thanks!

**Comment by szpajder** [comment]
You have specified the 119 MHz frequency as an integer number. The program interprets integers as a frequencies given in hertz, rather than megahertz. Hence you have a frequency of 119 Hz in your scanning list (which is more or less 0 MHz).
